.changeset/perfect-coats-train.md

```diff
@@ -0,0 +1,60 @@
+---
+'@backstage/plugin-catalog': patch
+---
+
+Adds a new `SystemDiagram` component to visually map all elements in a system.
+
+To use this new component, a new system entity page would need to be added to the `packages/app/src/components/catalog/EntityPage.tsx` file.
```

Comment:
now with this merged w/ the system tables PR #4392 technically, this changelog could assume the other system tables changelog has been implemented, and thus simplify this to ONLY talk about adding the tab as a recommendation (though it's left up to the implementer team where they want to put the component); thoughts?